tests/Unit/ActivityHelper/LogNotificationTest.php

```diff
@@ -35,7 +35,7 @@ protected function setUp(): void
 
     public function testLogsStatus(): void
     {
-        $this->assertDatabaseHas('activity_log', [
```

Comment:
if i do, i run back into that problem i showed you where it kept inferring the table name as 'activities' and trying to read from that, i thought it would be easier to just rename the tests

Reply:
on other tests i mean

Reply:
Based on the base Spatie activity, I disagee.

![image](https://github.com/user-attachments/assets/40a55902-8b45-4f46-b4c6-36a0a99e8c4f)